docs/.vitepress/i18n/component/demo-block.json

```diff
@@ -1,8 +1,9 @@
 {
   "en-US": {
     "view-source": "View source",
-    "edit-on-github": "Edit on Github",
     "edit-in-editor": "Edit in PlayGround",
+    "edit-on-github": "Edit on Github",
+    "edit-in-codepen": "Edit in Codepen.io",
```

Comment:
We don't need codepen anymore.

Reply:
Agreed.

Reply:
In face, i think it is generated by crowdin. Because we have it in crowdin.